### DISCUSSION TITLE
xmake怎样添加src下子工程里面的.h文件

### DISCUSSION THREAD

**Discussion by Shao-junliang** [xmake-io/xmake#5166] [Q&A]
![image](https://github.com/xmake-io/xmake/assets/63970570/e073c674-ba7a-4b96-b11f-c27820c0ed22)
fatal error C1083: 无法打开包括文件: “cuda_driver_api.h”: No such file or directory
这种情况应该怎样写lua，部分两个target写的情况下，怎样写在一个target下面

**Comment by waruqi** [comment]
add_includedirs 自己对下路径。
